src/cljs/test/broadfcuitest/strip_comment_leaders.cljs

```diff
@@ -0,0 +1,37 @@
+(ns broadfcuitest.strip-comment-leaders
+  (:require
+   [cljs.test :refer [deftest is testing]]
+   [broadfcui.page.method-repo.create-method :refer [strip-comment-leaders]]
+   [broadfcui.utils :as utils]
+   ))
+
+
+(deftest check-strip-comment-leaders
```

Comment:
Is this a valid use case, where the hashes are indented?
```
     ## A
     ## B
          ## C
```

Reply:
Yeah, I suppose that's valid. I'll handle that and add a test.